packages/docusaurus-theme-classic/src/theme/CodeBlock/WordWrapButton/styles.module.css

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+.wordWrapButton {
+  right: calc(
```

Comment:
I'd like the button to not decide where it will be rendered: that's the responsibility of the parent

Reply:
I don't like it either, I was thinking of grouping these buttons into one new div container, but it's not that easy because we would have to somehow get background of the current Prism color theme (need it for button background, not its container). Therefore `inherit` value won't work here, so maybe we dynamically set two new CSS vars, like as `--prism-background` and `--prism-color`? Will that be acceptable solution?

Reply:
Maybe this works? `usePrismTheme().plain.backgroundColor`

Reply:
I mean, in the case of button container, we need to set the `style` attribute for each button inside to set the proper background. I think it's kind of redundant overhead, which we can avoid by dynamically creating two CSS vars that also can re-use in other places where the `style` attribute is currently used.

![image](https://user-images.githubusercontent.com/4408379/162459100-57549bf9-e711-422a-808f-bf76980ba586.png)

Reply:
@lex111 FWIW, Prism also inlines their styles, so it won't be too harmful

Reply:
Just wouldn't want to overuse it, especially if there's a pretty appropriate alternative solution.  Beside that, using inline styles would require creating one specific prop `style` for each button component, which can also be avoided.

Reply:
Is this rule still necessary?

Looks to me it's not useful anymore => removing it and things keep working?